ykrt/src/compile/jitc_yk/codegen/x64/lsregalloc.rs

```diff
@@ -706,12 +706,11 @@ impl<'a> LSRegAlloc<'a> {
         for cnstr in &constraints {
             match cnstr {
                 RegConstraint::InputIntoReg(_, reg)
-                | RegConstraint::InputIntoRegAndClobber(_, reg) => avoid.set(*reg),
+                | RegConstraint::InputIntoRegAndClobber(_, reg)
```

Comment:
The only part of the PR I'm uncertain of is this part. So the actions for the constraints were just wrong altogether before and this part isn't much to do with floats?